engine/server/server.go

```diff
@@ -75,6 +79,13 @@ func startJob(model string, out string, jobs map[string]*os.Process) (string, er
 	if err != nil {
 		return id, err
 	}
+	
+	// Change model path to individual simulation's modified xml
+	model = xmlConvert(model, out, params)
```

Comment:
I think `xmlConvert` should have return type `(string, error)`, see comment below